ingestion-beam/src/test/java/com/mozilla/telemetry/RepublisherMainTest.java

```diff
@@ -37,10 +37,10 @@ public void testDebugDestination() throws Exception {
     String input = inputPath + "/*.ndjson";
     String output = outputPath + "/out";
 
-    Republisher
-        .main(new String[] { "--inputFileFormat=json", "--inputType=file", "--input=" + input,
-            "--outputFileFormat=json", "--outputType=file", "--enableDebugDestination",
-            "--debugDestination=" + output, "--outputFileCompression=UNCOMPRESSED" });
+    Republisher.main(new String[] { "--inputFileFormat=json", "--inputType=file",
+        "--input=" + input, "--outputFileFormat=json", "--outputType=file",
+        "--enableDebugDestination", "--debugDestination=" + output,
+        "--outputFileCompression=UNCOMPRESSED", "--errorOutputType=stderr" });
```

Comment:
`errorOutputType` defaults to `pubsub` and previously, these tests ran without actually calling any code for the error output, bit looks like 2.13.0 changes something internally in Beam such that the error output is accessed for display data, leading to exceptions (since this test isn't actually set up to provide everything a pubsub-based error output needs).

We now add `"--errorOutputType=stderr"` explicitly to many tests to avoid this scenario.